fe/fe-core/src/main/java/com/starrocks/scheduler/TaskRun.java

```diff
@@ -122,6 +124,31 @@ public ConnectContext getRunCtx() {
     }
 
     public TaskRunStatus getStatus() {
+        if (status == null) {
+            return null;
+        }
+        switch (status.getState()) {
+            case RUNNING:
+                if (runCtx != null) {
+                    StmtExecutor executor = runCtx.getExecutor();
+                    if (executor != null) {
+                        Long jobId = executor.getCoordinator().getJobId();
+                        if (jobId != -1) {
+                            InsertLoadJob job = (InsertLoadJob) GlobalStateMgr.getCurrentState()
+                                    .getLoadManager().getLoadJob(jobId);
+                            int progress = job.getProgress();
```

Comment:
Is there any change that job is null?

Reply:
the defautl value is -1 when it's not uninitialized